internal/spentcalories/spentCalories.go

```diff
@@ -14,71 +16,82 @@ const (
 )
 
 func parseTraining(data string) (int, string, time.Duration, error) {
-	// ваш код ниже
+	slice := strings.Split(data, ",")
+	if len(slice) != 3 {
+		return 0, "", 0, fmt.Errorf("the number of parameters shoud be 3 ")
+	}
+
+	steps, err := strconv.Atoi(slice[0])
+	if err != nil {
+		return 0, "", 0, fmt.Errorf("can not convert string into integer")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, "", 0.0, fmt.Errorf("conversion error: %w", err)
```